tox.ini

```diff
@@ -23,9 +23,14 @@ commands =
 [testenv:type]
 description = run static type checking
 basepython = python3
-usedevelop = True
 commands = mypy -p snorkel
 
+[testenv:coverage]
```

Comment:
Note (earlier in file): typo line 12

Reply:
what kind of coverage tests are these?  Redundant with codecov?

Reply:
Not a typo: https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints

Reply:
This command (`pytest --cov=...`) generates the `.coverage` file. The command `codecov` just looks for a file called `.coverage` and uploads it. See https://github.com/codecov/example-python/blob/master/.travis.yml